docs/zkapps/faq.mdx

```diff
@@ -111,6 +111,23 @@ this.num.assertEquals(currentState);
 
 This ensures that the transaction fails if the value of the field in question has changed.
 
+### How do I restrict a provable method so that it can be invoked only by a specific address?
+
+There are a number of ways to accomplish this:
+- Provably add a signed child account update. Then, the method can be invoked only with a signature from the predefined address.
```

Comment:
```suggestion
- Provably add a signed child account update. Then, you can invoke the method only with a signature from the predefined address.
```

Reply:
I was looking at open PRs and noticed we let this one get stale... do we want to discuss? @garwalsh